cmd/notation/cert.go

```diff
@@ -8,6 +8,7 @@ import (
 	"time"
 
 	"github.com/notaryproject/notation-core-go/x509"
+	configutil "github.com/notaryproject/notation-go/config"
```

Comment:
Let's keep `github.com/notaryproject/notation-go/config` as `config`, and rename `github.com/notaryproject/notation/pkg/config` as `github.com/notaryproject/notation/pkg/configutil`.

Reply:
Done and tested again.